core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -143,7 +144,8 @@ object Boot extends App with LazyLogging {
 
     val genomicsServiceConstructor: (UserInfo) => GenomicsService = GenomicsService.constructor(slickDataSource, gcsDAO, userDirDAO)
     val statisticsServiceConstructor: (UserInfo) => StatisticsService = StatisticsService.constructor(slickDataSource, gcsDAO, userDirDAO)
-    val methodRepoDAO = new HttpMethodRepoDAO(conf.getConfig("methodrepo").getString("server"))
+    val agoraConfig = conf.getConfig("methodrepo")
+    val methodRepoDAO = new HttpMethodRepoDAO(agoraConfig.getString("server"), agoraConfig.getString("path"))
```

Comment:
I split the Agora URL config into 2 components:
```
methodrepo {
  server = "https://agora.dsde-dev.broadinstitute.org:443"
  path = "/api/v1"
}
```
This is because the Agora status endpoint just lives at /status, so I need to be able to get the base URL. I think this means an update to https://github.com/broadinstitute/firecloud-develop, I'll open a PR for that shortly.